tools/reassign-alias.js

```diff
@@ -0,0 +1,50 @@
+const elastic = require('../lib/elastic')
+
+function createErrorMessage (value) {
+  console.log(`You have not provided a value for ${value}. Please provide a value for ${value} with the --${value} option before continuing`)
```

Comment:
**Suggestion:** I think you missed converting this `console.log` to a return of a string.

```suggestion
  return `You have not provided a value for ${value}. Please provide a value for ${value} with the --${value} option before continuing`
```

Reply:
ahh yes! 🤦‍♀️